pkgs/development/python-modules/schwifty/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, iso3166
+, pycountry
+}:
+
+buildPythonPackage rec {
+  pname = "schwifty";
+  version = "2022.6.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-MekF96K8IPjop5764Oq6ZcvKJOTc1Qg/gV5Dz2iacBk=";
+  };
+
+  propagatedBuildInputs = [ iso3166 pycountry ];
+
```

The project seems to be using `pytest` to run the tests. `pythonImportsCheck` is an additional safe-guard.

```suggestion

  checkInputs = [
    pytestCheckHook
  ];

  pythonImportsCheck = [
    "schwifty"
  ];
```

---

It could also be required that you patch out the coverage part of the tests. `tox.ini` is mentioning `pytest-cov`.